src/utils.ts

```diff
@@ -0,0 +1,9 @@
+export const capitalizeFirst = (str: string) => {
+    return str.charAt(0).toUpperCase() + str.slice(1);
+  };
```

Comment:
In one of Tim's PRs we have removed this in favor of CSS text-transform: capitalize. I think he created a class called ".capitalize" that is then assigned where needed. 

My thinking is to use what already comes out of box in CSS, but if there is a need to have this as JS function, we can keep it. :)

Reply:
Makes sense!